### PR TITLE
doc: fix typos and grammar nits

### PR DESCRIPTION
**Pull request by coryan**

### CODE REVIEW

File: Versioning.md [hunk 1]
```diff
@@ -19,38 +19,38 @@ must remain backwards compatible. Internal types are allowed to break.
...
 
 ## Policy
 
 * Release versions will follow [SemVer 2.0](https://semver.org/).
-* Only single source package containing api and sdk for all signals would be released as part of the new GitHub release.
-* There would be source package releases for api and sdk. There won't be separate releases for the signals. The release version numbers for api and sdk will not be in sync with each other. As there would be more frequent changes expected in sdk then in api.
-* Experimental releases: New (unstable) telemetry signals and features will be introduced behind feature flag using C macro define.
+* Only a single source package containing both the api and sdk for all signals will be released as part of each GitHub release.
+* There will be source package releases for api and sdk. There won't be separate releases for the signals. The release version numbers for api and sdk will not be in sync with each other. As there would be more frequent changes expected in sdk than in the api.
```
Comment (coryan) [inline Versioning.md:30]: This seems repetitive.

File: Versioning.md [hunk 2]
```diff
@@ -19,38 +19,38 @@ must remain backwards compatible. Internal types are allowed to break.
...
+* ABI stability is guaranteed for the API.
+* ABI stability is not guaranteed for the SDK. In case of ABI breaking changes, instead of bumping up the major version, a new `inline namespace` version will be created, and both old API and new API would be made available simultaneously.
 
 ## Policy
 
 * Release versions will follow [SemVer 2.0](https://semver.org/).
-* Only single source package containing api and sdk for all signals would be released as part of the new GitHub release.
-* There would be source package releases for api and sdk. There won't be separate releases for the signals. The release version numbers for api and sdk will not be in sync with each other. As there would be more frequent changes expected in sdk then in api.
-* Experimental releases: New (unstable) telemetry signals and features will be introduced behind feature flag using C macro define.
+* Only a single source package containing both the api and sdk for all signals will be released as part of each GitHub release.
```
Comment (coryan) [inline Versioning.md:29]: API and SDK are styled inconsistently, consider using the same case (`api` vs. `API`) throughout the document. Consider also defining or referencing a definition of what the API vs. the SDK is.

File: Versioning.md [hunk 3]
```diff
@@ -19,38 +19,38 @@ must remain backwards compatible. Internal types are allowed to break.
 
 ### ABI Stability
 
-Refer [ABI Policy](./docs/abi-policy.md) for more details. To summarise
-* ABI stability is guaranted for API.
-* ABI stability is not guaranteed for SDK. C++ In case of ABI breaking changes, instead of bumping up the majore version, new `inline namespace` version will be crearted, and both old API and new API would be made available simultaneously.
+Refer to the [ABI Policy](./docs/abi-policy.md) for more details. To summarise
+* ABI stability is guaranteed for the API.
+* ABI stability is not guaranteed for the SDK. In case of ABI breaking changes, instead of bumping up the major version, a new `inline namespace` version will be created, and both old API and new API would be made available simultaneously.
```
Comment (coryan) [inline Versioning.md:24]: It is ambiguous what these "ABI breaking changes" are referring to. Are these ABI breaking changes to the SDK?  or ABI breaking changes to the API?

Is the plan really to keep two copies of the code (approximately) forever after an ABI breaking change?

Comment (lalitb) [reply Versioning.md:24]: `ABI breaking changes` refers to SDK here. Will reword it properly. 

> Is the plan really to keep two copies of the code (approximately) forever after an ABI breaking change?

Yes, that's the plan for now, but will be revisiting this before going for GA.

File: Versioning.md (hunk 2 above)
Comment (lalitb) [reply Versioning.md:29]: Will add that. Thanks for noticing it.

File: Versioning.md (hunk 1 above)
Comment (lalitb) [reply Versioning.md:30]: Correct. Need to merge both the points.

**Review by lalitb — APPROVED** [review]
Thanks for fixing the typos and grammar. Time for me to use a spell check extension for VS Code : )

**Comment by lalitb** [comment]
Merging this, as it is documentation nit fix. If any comments, please put in the review comment, and can be addressed separately.

**Pull request merged**